public/app/core/components/AppChrome/News/NewsItem.tsx

```diff
@@ -0,0 +1,138 @@
+import { css } from '@emotion/css';
+import React, { useEffect } from 'react';
+import { useAsyncFn } from 'react-use';
+
+import { GrafanaTheme2, textUtil, dateTimeFormat } from '@grafana/data';
+import { useStyles2, LoadingPlaceholder } from '@grafana/ui';
+
+import { fetchNews } from './api';
+
+export function NewsItem() {
```

Comment:
Any reason why this is not using the same component as the new panel? we can technically just render the news panel as well via PanelRenderer

Reply:
but it's probably just better to refactor our NewsItem and reuse it in both, we should definitely not have two identical components :)

Reply:
I see you ask where to place it, I think place the shared component in the news panel and reference it from here

Reply:
Thanks @torkelo , I will update it. If I understand correctly, there is no strict rule about cross imports, this can happen between app/core/components and app/plugins?

Reply:
yea, we are not super strict there